config/v1/feature_gates.go

```diff
@@ -341,4 +341,14 @@ var (
 		ResponsiblePerson:   "padillon",
 		OwningProduct:       ocpSpecific,
 	}
+
+	FeatureGateCollectionProfiles = FeatureGateName("CollectionProfiles")
```

Comment:
Maybe prefix with `metrics` to make it more explicit?
```suggestion
	FeatureGateMetricsCollectionProfiles = FeatureGateName("MetricsCollectionProfiles")
```